build.assets/Makefile

```
@@ -11,13 +11,17 @@ NOROOT=-u $$(id -u):$$(id -g)
 KUBECONFIG ?=
 TEST_KUBE ?=
 
+SHELL=bash
+
 OS ?= linux
 ARCH ?= amd64
-RUNTIME ?= go1.15.5
+RUNTIME ?= go1.16.2
 
 UID := $$(id -u)
 GID := $$(id -g)
 
+BORINGCRYPTO_RUNTIME=$(RUNTIME)b7
```

Move this up below `RUNTIME`

```suggestion
RUNTIME ?= go1.16.2
BORINGCRYPTO_RUNTIME=$(RUNTIME)b7

UID := $$(id -u)
GID := $$(id -g)
```